iree/test/e2e/linalg_ext_ops/BUILD

```diff
@@ -72,7 +72,8 @@ iree_check_single_backend_test_suite(
         # keep sorted
         [
             "reverse.mlir",
-            "top-k.mlir",
+            # Top-k test disabled due to miscompile on vulkan.
+            #    "top-k.mlir",
```

Comment:
THis is probaby going to fail presubmit if not added to the "explicit" list.

Reply:
yes it did, updated it